### DISCUSSION TITLE
wire:click show error JS

### DISCUSSION THREAD

**Discussion by utqinadhif** [livewire/livewire#6533] [7. Bugs (archived)] [answered]
![image](https://github.com/livewire/livewire/assets/10701527/91b4f1d6-f253-4a78-b175-68dfe2d72d91)

i try with upgrade from v2 to v3, but error on wire:click

**Comment by utqinadhif** [comment]
After test and try, the error show if element using @forelse , if i use foreach, error js not shown, why?

**Comment by utqinadhif** [comment]
Please fix if my component using @forelse

**Comment by joshhanley** [comment, marked as answer]
@utqinadhif this has been fixed and released in 3.0.1. Can you please upgrade and test it, and let us know if there are any issues. Thanks!

**Comment by utqinadhif** [comment]
Nice, thanks
